src/Data/Bin/Ordering.agda

```diff
@@ -0,0 +1,790 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Ordering on Bin and certain its properties.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Bin.Ordering where
+
+open import Data.Bin.Base
+open import Data.Bin.Properties
+open import Data.Nat as ℕ using (ℕ; z≤n; s≤s)
+open import Data.Nat.Properties as ℕₚ using (m+n∸m≡n; m+n∸n≡m)
+open import Data.Product using (_,_; proj₁; proj₂)
+open import Data.Sum
+open import Function
+open import Level using () renaming (zero to 0ℓ)
+open import Relation.Binary
+open import Relation.Binary.PropositionalEquality as P
+open import Relation.Binary.PropositionalEquality.Core using (≢-sym)
+import Relation.Binary.Reasoning.Base.Triple as InequalityReasoning
+open import Relation.Nullary using (¬_; yes; no)
+open import Relation.Nullary.Negation using (contradiction)
+
+open Bin
+
+-- The ordering relation
+
+infix 4 _<_ _>_ _≤_
```

Comment:
You're missing some of the relations here.

Reply:
> I agree that parantheses should be avoided if possible, but in this case I still think they're 
> more readable than the alternatives. There shouldn't be any examples of where we use 
> unbalanced parentheses in names. Can you point out where we do so and I'll change 
> them?

I see the functions ``2[1+-as∘, 1+[2-as∘``. They are applied like this:
```
  suc (double x)   ≈⟨ sym (1+[2-as∘ x) ⟩
  1+[2 x ]         ∎
```
Now I rename them to ``2[1+_]-as∘, 1+[2_]-as∘``.
It this all right? 


>> infix 4 _<_ _>_ _≤_

> You're missing some of the relations here.

I add there ``_≮_ _≯_ _≰_ _≱_``.